roles/edpm_nova/molecule-requirements.txt

```diff
@@ -0,0 +1,8 @@
+ansible-core<2.12.0
```

Comment:
Reqs should reside in one place, if possible.

Reply:
yep as noted its not possibel currently as we cannot use the same version fo ansible due to python path issues

https://github.com/openstack-k8s-operators/edpm-ansible/pull/215#discussion_r1263630455

i do not have time to try and unify them in the short term.